### DISCUSSION TITLE
Edit the export of the project

### DISCUSSION THREAD

**Discussion by Manesx** [godotengine/godot-proposals#8726] [Engine Core]
I suggest storing the resources after export in a single file without encryption. Information on each attached file (path, offset, size) should be stored in a single file that can be encrypted. When opening a project, the godot engine should access this file for the necessary file.

**Comment by Calinou** [comment]
Godot already does this on desktop platforms, iOS and Web. For Android, this is already being tracked in https://github.com/godotengine/godot-proposals/issues/946.
